README.md

```diff
@@ -209,7 +207,7 @@ RESTful | [keymatch_model.conf](https://github.com/casbin/node-casbin/blob/maste
 Deny-override | [rbac_model_with_deny.conf](https://github.com/casbin/node-casbin/blob/master/examples/rbac_with_deny_model.conf)  | [rbac_policy_with_deny.csv](https://github.com/casbin/node-casbin/blob/master/examples/rbac_with_deny_policy.csv)
 Priority | [priority_model.conf](https://github.com/casbin/node-casbin/blob/master/examples/priority_model.conf)  | [priority_policy.csv](https://github.com/casbin/node-casbin/blob/master/examples/priority_policy.csv)
 
-## Our adopters
+## Our adapters
```

Comment:
This is not typo. ``adopter`` means the products that ``adopt`` our framework, not the ``adapters`` for storage. So please revert it.